### ISSUE TITLE
symlink for 10.0 docs

### ISSUE THREAD

**Issue by patriknw** [akka/akka-http#1133]
should we have the same kind of symlinks as for Akka? For example, I expected this to work: http://doc.akka.io/docs/akka-http/10.0/security/2017-05-03-illegal-media-range-in-accept-header-causes-stackoverflowerror.html

we don't want links directly to 10.0.6

**Comment by jlprat** [comment]
We currently have the `current` symlink, but I think we should also have this other you mention :+1:

**Comment by 2m** [comment]
Should we also consider not hosting browsable documentation of old patch releases and host archived zips instead?

**Comment by patriknw** [comment]
> not hosting browsable documentation of old patch releases and host archived zips instead

that is a possibility if the canonical refs don't work

we will typically have 2 supported versions (e.g. 10.0 and 10.1) and it's sometimes good to link to the specific version but don't care about the patch

**Comment by ktoso** [comment]
Yes we should have the `10.0` symlink; I think it's an omission that we don't, not something on purpose.

I'm not sure about the not-at-all-hosting old versions hm... Let's see if the canonical links will get us googlability first. Would not mind removing "super old ones", like 1.0 (http/streams) though, but 2.4 should docs stay around.

**Comment by patriknw** [comment]
I agree that we prefer to get the canonical links working.

> but 2.4 should docs stay around

yes, what I mean is that it would be possible to archive old patch versions, e.g. 2.4.3, but latest 2.4.18 (2.4) must of course exist

For EOL versions we can archive all of them if needed.

**Comment by ktoso** [comment]
> For EOL versions we can archive all of them if needed.

Exactly, we're in agreement then :)

**Comment by jonas** [comment]
It would be good to also add/maintain a list of old Akka HTTP doc versions in http://doc.akka.io/robots.txt. Right now it only contains Akka versions.

**Comment by raboof** [comment]
The reason we don't have all old versions in `robots.txt` is that we're trying to use canonical link headers (https://support.google.com/webmasters/answer/139066?hl=en#6) so google doesn't ignore the pages, but redirects the 'seo juice' to the current version of that page. To be honest we're not seeing that take effect very convincingly yet but perhaps it needs some tweaking or just giving Google some time to adjust.

I added the 2.4 akka pages to robots.txt because those pages actually have a 'canonical' tag in the HTML, but it's pointing to the wrong place :(.

For 'super old' pages we should perhaps try to 301 redirect to something newer?

**Comment by jrudolph** [comment]
I agree, we need to make a plan which old links to support and by which measures.
